tests/unit/style/properties/serialization.rs

```diff
@@ -1139,3 +1148,26 @@ mod shorthand_serialization {
 
     }
 }
+
+#[test]
+fn test_border_spacing() {
```

Comment:
I think this belongs to `tests/unit/style/parsing/border.rs`. This is a parsing test normally. We don't check if the values are correct here. Could you move it there please?